packages/access-client/src/agent.js

```diff
@@ -704,7 +754,7 @@ export class Agent {
    */
   async #waitForDelegation(opts) {
     const ws = new Websocket(this.url, 'validate-ws')
-    await ws.open()
+    await ws.open(opts)
```

Comment:
`authorize()` would hang here because abort signal not passed to this

Reply:
(and even if it were, it wouldn't respect it without the other change I put in there)